src/libraries/System.Windows.Extensions/src/System/Media/SoundPlayer.cs

```diff
@@ -271,7 +274,9 @@ private void LoadSync()
                 _stream = webResponse.GetResponseStream();
             }
 
-            if (_stream.CanSeek)
+            // DO NOT assert - NRE is expected for null stream
+            // See SoundPlayerTests.Load_NullStream_ThrowsNullReferenceException
```

Comment:
It'd be worth opening an issue for this to decide separately whether we should throw a better exception in that case.